tools/deployment/com.facebook.osqueryd.plist

```diff
@@ -9,7 +9,7 @@
   <key>Label</key>
   <string>com.facebook.osqueryd</string>
   <key>Program</key>
-        <string>/usr/local/bin/osqueryd</string>
+    <string>/usr/local/bin/osqueryd --flagfile="/var/osquery/osquery.flags"</string>
```

Comment:
@theopolis Makes sense to me. I kept it simple, but didn't check out the spec. Certainly should have.